appveyor.yml

```diff
@@ -131,32 +117,18 @@ install:
   # Install mingwpy toolchain
   #
   - ECHO "Installing MinGW toolchain for Python"
-  # This would have been easier, but was resulting in issue #913
-  #- cmd: "%PYTHONHOME%\\Scripts\\pip.exe install -i https://pypi.anaconda.org/carlkl/simple mingwpy
-
-  # Reference: https://bitbucket.org/carlkl/mingw-w64-for-python/downloads/mingwpy-2015-04-readme.md
-
-  # List contents of C:\ to help debug caching of the artifacts
-  - DIR C:\
-
-  - ps: $dwntool=New-Object System.Net.WebClient
-
-  - ECHO "Installing MinGW-w64 for Python via archive %mingwpy_archive_name%:"
-  - ps: if (-Not (Test-Path "C:\\$env:mingwpy_archive_name")) { $dwntool.DownloadFile("https://bitbucket.org/carlkl/mingw-w64-for-python/downloads/$env:mingwpy_archive_name", "C:\\$env:mingwpy_archive_name") } else { Write-Host "Found" $env:mingwpy_archive_name "in cache." }
-  - 7z x C:\%mingwpy_archive_name% -oC:\ > null
-
-  - ECHO "Installing MinGW-w64 Python 2.7.x libs via archive %libpython_archive_name% into %PYTHONHOME%:"
-  - ps: if (-Not (Test-Path "C:\\$env:libpython_archive_name")) { $dwntool.DownloadFile("https://bitbucket.org/carlkl/mingw-w64-for-python/downloads/$env:libpython_archive_name", "C:\\$env:libpython_archive_name") } else { Write-Host "Found" $env:libpython_archive_name "in cache." }
-  - 7z x C:\%libpython_archive_name% -o%PYTHONHOME%\ > null
-
-  - set PATH=C:\mingwpy\bin;%PATH%
-
+  # Install the version of mingwpy toolchain that generates targets compatible
+  # with the installed python (Win32 vs. Win64). The compiler tools are
+  # installed into %PYTHONHOME%/Scripts/
+  - cmd: "%PYTHONHOME%\\Scripts\\pip.exe install -i https://pypi.anaconda.org/carlkl/simple mingwpy==0.1.0b3"
```

Comment:
@scottpurdy, I picked the current version of mingwpy ("0.1.0b3") in the installation. Is it better to remove the specific version, and let CI install the latest available version of the mingwpy toolchain?